.github/workflows/ci.yaml

```diff
@@ -11,4 +11,4 @@ jobs:
     # conform since this is a collaborative repo with an external group.
     uses: nextstrain/.github/.github/workflows/pathogen-repo-ci.yaml@v0
     with:
-      build-args: test_target
+      build-args: --configfile config/gisaid.yaml -pf test_target
```

Comment:
Urgh. Thanks for noticing - this would have been a pain to solve post-merge!

I like the explicitness of providing a `--configfile` for each run. But do I like it enough to make changes across 3 extra repos? Probably. Will think about it some more.

Reply:
(Only noticed because we recently went through this in https://github.com/nextstrain/avian-flu/pull/17).

Reply:
PRs made to update CI in
* [augur](https://github.com/nextstrain/augur/pull/1552)
* [conda-base](https://github.com/nextstrain/conda-base/pull/86)
* [docker-base](https://github.com/nextstrain/docker-base/pull/224)